cg/server/invoices/templates/invoices/new.html

```diff
@@ -83,15 +87,19 @@ <h1>Make new invoice ({{record_type}})</h1>
 
 
 {% block scripts %}
-<script language="JavaScript">
-  function toggle() {
-    checkboxes = document.getElementsByName('records');
-    for(var checkbox in checkboxes) {
-      checkboxes[checkbox].checked = !checkboxes[checkbox].checked;
+
+<script language="JavaScript"> 
+    function check_all() {
+      checkboxes = document.getElementsByName('records');
+     
+      for(var checkbox in checkboxes) {
+
+        checkboxes[checkbox].checked = !checkboxes[checkbox].checked;
+      }
+      
+      checkTotal()
     }
-  checkTotal()
-  }
-</script>
+  </script>
 
 <script language="JavaScript">
```

Comment:
Even though this works perfectly, there is no need to wrap all javascript functions in separate `script` tags. You could wrap them in just one.